stable/uchiwa/templates/deployment.yaml

```diff
@@ -18,24 +18,28 @@ spec:
       - name: uchiwa
         image: "{{ .Values.image }}:{{ .Values.imageTag }}"
         imagePullPolicy: {{ .Values.pullPolicy }}
-{{ toYaml .Values.resources | indent 8 }}
+        resources:
+{{ toYaml .Values.resources | indent 10 }}
```

Comment:
Actually it may be the Sensu chart that is screwed up. I believe K8s 1.5.1 is now being more strict about extraneous fields.